src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rental = 40;
+  const price = rental * days;
+
+  if (days < 3) {
+    return price;
+  }
+
+  if (days < 7) {
+    return price - 20;
+  }
+
+  if (days >= 7) {
+    return price - 50;
```

Comment:
You can remove the `if` condition here since it's guaranteed that `days` is at least 7 due to the previous conditions. This simplifies the logic and adheres to checklist items 4 and 5.